src/CMakeLists.txt

```diff
@@ -232,7 +232,25 @@ install(FILES
     ${CMAKE_BINARY_DIR}/exiv2lib_export.h
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/exiv2)
 
-install(EXPORT exiv2Config DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2")
+# CMake configs
+set(export_dir "${CMAKE_BINARY_DIR}/export")
```

Comment:
[suggestion] Since the motto in modern CMake is to use as less variables as possible, I would try to avoid the addition of the `export_dir` variable. I tried this and it works too:

```
# CMake configs
include(CMakePackageConfigHelpers)
write_basic_package_version_file("exiv2ConfigVersion.cmake" COMPATIBILITY AnyNewerVersion)

configure_package_config_file(
    "${CMAKE_SOURCE_DIR}/cmake/exiv2Config.cmake.in"
    "exiv2Config.cmake"
    INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2"
)

install(FILES
    "${CMAKE_CURRENT_BINARY_DIR}/exiv2Config.cmake"
    "${CMAKE_CURRENT_BINARY_DIR}/exiv2ConfigVersion.cmake"
    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2"
)
```

However it is a minor cosmetic issue. I'll leave to you the final decision.

Thanks for this great contribution! I new that CMake could provide information about the 3rd party dependencies to the downstream projects, but one had to know the dependencies to include those `find_package` calls in the downstream projects. I never investigated how to propagate such information. It's good to be learning CMake stuff from other people in Exiv2 :heart_eyes:

Reply:
I can change that. I'd rather stick to the style the project maintains.

You're very welcome. It took me a long time to find out how to provide dependencies downstream. Much of the credit goes to Pablo Arias who wrote [a great post](https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/) laying out good library structure and demonstrating `find_dependency`. It's still not perfect (the work around for optional deps is particularly annoying), but it's much better than pushing the work off onto downstream projects.